src/git.rs

```diff
@@ -65,10 +64,22 @@ impl GitConfig {
     }
 }
 
+fn git_credentials_callback(
+    _url: &str,
+    username_from_url: Option<&str>,
+    _allowed_types: git2::CredentialType,
+) -> Result<git2::Cred, git2::Error> {
+    let mut priv_key =
+        dirs::home_dir().ok_or_else(|| git2::Error::from_str("$HOME was not set"))?;
+    priv_key.push(".ssh/id_rsa");
+    Cred::ssh_key(username_from_url.unwrap_or("git"), None, &priv_key, None)
+}
+
 pub(crate) fn create(project_dir: &Path, args: GitConfig) -> Result<String> {
     let temp = Builder::new().prefix(project_dir).tempdir()?;
     let config = Config::default()?;
     let remote = GitRemote::new(&args.remote);
+    let is_http = args.remote.to_string().contains("http");
```

Comment:
suggestion: to harden that code a bit more, I'd suggest to check if the url starts with that protocol, since contains can yield false positives, like a repo that has `http` in it's name like `git@github.com:hyperium/http.git`

Reply:
Good point. Switched to checking for `ssh://`